lib/ModelMixins/CatalogMemberMixin.ts

```diff
@@ -235,3 +240,16 @@ export const getName = action((model: BaseModel | undefined) => {
     "Unknown model"
   );
 });
+
+function mustacheNestedJsonObject(obj: JsonObject, view: any) {
```

Comment:
It is pretty obvious, but you still might want to add a one line comment for this function.

Reply:
Done!